TemplateFiles/DeploymentFiles/AzTSSetup.ps1

```diff
@@ -398,6 +410,11 @@ function Install-AzSKTenantSecuritySolution
                      $TemplateParameters.Add("FrontDoorEndpointSuffix", ".azurefd.us")
                      $TemplateParameters.Add("WebAppEndpointSuffix", ".azurewebsites.us")
                  }
+                 elseif($AzureEnvironmentName -eq 'AzureChinaCloud')
+                 {
+                     $TemplateParameters.Add("FrontDoorEndpointSuffix", ".azurefd.cn") 
+                     $TemplateParameters.Add("WebAppEndpointSuffix", ".azurewebsites.cn")
```

Comment:
is this correct?

Reply:
I have changed the value for WebAppEndpointSuffix to **.chinacloudsites.cn** .